docs/user/whats-new.asciidoc

```diff
@@ -1,157 +1,250 @@
 
 [role="screenshot"]
-image::images/improved-errors.png[A screenshot of an improved error message, width=50%]
+image::images/esql-validation.png[An image of {esql} validation in action.]
 
 [discrete]
-=== Discover
+==== Color all terms in new color mapping
+
+In our latest update, based on early feedback, we've enhanced the user experience with our new color mapping feature introducing a more intuitive default option. Now users can either assign a single color or enable color looping for unassigned terms, providing greater flexibility and a cleaner interface for term assignments. Additionally we've eliminated the maximum limit on assignments, allowing for more comprehensive customization. It's important to note that looping colors or using more than 10 colors can be effective in certain scenarios. However, we recommend limiting the number of colors used to prevent potential misinterpretation of your charts. This update aims to make color mapping more user-friendly and adaptable to your needs.
+
+[role="screenshot"]
+image::images/color-mapping-enhanced.png[An image of the color mapping feature applied to a vertical bar chart.]
+
+[role="screenshot"]
+image::images/color-mapping-1.png[An image of the color palette options, width=60%]
 
 [discrete]
-==== Improved long field names handling in {kib}
-Long field names are very normal in Observability and Security datasets. That’s why we adapted multiple elements in Discover, Dashboards, Maps, and Lens such as field selectors, table headers, filter pills, and chart tooltips amongst others to handle long field names. For example, you will notice it when you select a field to set some filters or when you mouse over a chart.
+==== Visualizing an {esql} query in Observability AI assistant
+
+The observability AI assistant in 8.13 comes with great improvements in query generation and performance. Users can now visualize the generated {esql} queries, edit them using the inline editing flyout, and embed them in a dashboard.
 
 [role="screenshot"]
-image::images/long-field-names.png[A screenshot of the improved long field name handling in {kib}]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltd6737b8e5633e948/viz-esql.gif[A gif of an {esql} query in the AI assistant.]
 
 [discrete]
-==== Improved search for field names by handling spaces like wildcards
-To improve data exploration, we improved the search within the field list by allowing users to do a more flexible search in the fields sidebar with terms containing spaces.
+==== Cross cluster queries support in {esql} and in {kib}
+
+Cross Cluster Queries are now supported in {esql} and in {kib} the feature has been introduced for both validation and autocomplete. The autocomplete feature will show some documentation about the specific settings when navigating the suggestions.
```

Comment:
Per https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide we use the hyphen for "cross-cluster search" and "cross-cluster replication", so I'd assume this would be the same. Either way, the treatment here should match the section label.

```suggestion
Cross-cluster queries are now supported in {esql} and in {kib} the feature has been introduced for both validation and autocomplete. The autocomplete feature will show some documentation about the specific settings when navigating the suggestions.
```